Lib/glyphsLib/classes.py

```diff
@@ -2747,6 +2772,9 @@ def __init__(self, path=None):
                 p.parse_into_object(self, fp.read())
             self.filepath = path
 
+        for master in self.masters:
+            master.font = self
```

Comment:
Actually there is already a mechanism that links the masters to the font using the `parent` property, which is set in `FontMasterProxy`here:
https://github.com/punchcutter/glyphsLib/blob/c089739b2b67a12436291df38f5126fe852f9f7a/Lib/glyphsLib/classes.py#L402

Unfortunately the `parent` property is not documented at http://docu.glyphsapp.com/#gsfontmaster and doesn't work in Glyphs.app:
```
>>> print(Glyphs.font.masters[0].font().masters[0])
<GSFontMaster "Regular" width 100.0 weight 100.0>
>>> print(Glyphs.font.masters[0].font.masters[0])
Traceback (most recent call last):
  File "<string>", line 5, in <module>
AttributeError: 'objc.native_selector' object has no attribute 'masters'
>>> print(Glyphs.font.masters[0].parent.masters[0])
Traceback (most recent call last):
  File "<string>", line 5, in <module>
AttributeError: 'NSKVONotifying_GSFontMaster' object has no attribute 'parent'
```

Maybe the correct way (in the sense that is matches Glyphs' behaviour and minimizes the amount of code that you add) would be to have a `font()` method that returns `self.parent`?

Reply:
Yes, FontFontMasterProxy has it in the setter, but that's not being set at all.  
```
>>> font.masters
(<GSFontMaster "Light" width 100.0 weight 17.0>, <GSFontMaster "Regular" width 100.0 weight 90.0>, <GSFontMaster "Bold" width 100.0 weight 220.0>)
>>> font.masters[0]
<GSFontMaster "Light" width 100.0 weight 17.0>
>>> font.masters[0].parent
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'GSFontMaster' object has no attribute 'parent'
```